.github/workflows/ci.yml

```diff
@@ -949,6 +943,19 @@ jobs:
             "${DOCKER_IMAGE}" \
             "${TEST_SCRIPT}"
 
+  build_and_test_android:
+    needs: [setup, build_all]
+    if: fromJson(needs.setup.outputs.should-run)
+    uses: ./.github/workflows/build_and_test_android.yml
+    with:
```

Comment:
https://github.com/openxla/iree/actions/runs/4681088944

> [Invalid workflow file: .github/workflows/ci.yml#L949](https://github.com/openxla/iree/actions/runs/4681088944/workflow)
The workflow is not valid. In .github/workflows/ci.yml (Line: 949, Col: 11): Error from called workflow openxla/iree/.github/workflows/build_and_test_android.yml@0cf1846e10004aa798e9061e5128b5524d7a5d83 (Line: 135, Col: 26): Unrecognized named-value: 'martrix'. Located at position 1 within expression: martrix.target.label_exclude

Reply:
Fixed.